conf/server.yaml.template

```diff
@@ -0,0 +1,24 @@
+SERVER:
+  # Server hostname
+  # HOSTNAME: replace-with-satellite-hostname
+  # Path to private ssh key to be used when connecting via SSH.
+  # SSH_KEY:
```

Comment:
Please include `SSH_KEY_STRING` here as well

Reply:
I was not sure how one can provide a plain SSH Key String, `specially with new line chat or without` even I didnt try by myself in robottelo. But in CI it picks very well.

Still added for component owners on the basis of Go `find yourself`. :)

Reply:
The key string will likely be set via envvar, not directly in the file - either way, it can contain newlines as private keys should.